packages/transitive-overlay/src/TransitiveOverlay.story.tsx

```diff
@@ -337,3 +337,26 @@ export const OTP2ScooterItinerary = injectIntl(({ intl }) => (
   </>
 ));
 OTP2ScooterItinerary.decorators = [withMap([33.749, -84.388], 11)];
+
+export const ShowRouteArrows = (): JSX.Element => (
+  <>
+    <TransitiveOverlay
+      showRouteArrows
+      transitiveData={itineraryToTransitive(bikeOnlyItinerary, { companies })}
+      visible
+    />
+  </>
+);
+ShowRouteArrows.decorators = [withMap([45.520441, -122.68302], 16)];
+
+export const ColorOverride = (): JSX.Element => (
```

Comment:
Should we maybe consider making these storybook controls instead of new stories?

Reply:
I don't know what that is.